docs/chainlink-keepers/supported-networks.html

```diff
@@ -76,5 +77,21 @@
 
 {% endmarkdown %}
 
+{% markdown %}
+### Avalanche
+
+* **Payment Premium (paymentPremiumPPB):** <span>70</span>% gas premium
```

Comment:
Payment Premium for Avalanche Fuji and Mainnet is 40% - this is the final amount

Reply:
This is fixed.